src/cim/components/modals/AddHostModal.tsx

```diff
@@ -1,14 +1,34 @@
 import * as React from 'react';
 import { Flex, FlexItem, Modal, ModalBoxBody, ModalVariant, Radio } from '@patternfly/react-core';
-import { DownloadIso } from '../../../common';
+import { DownloadIso, PopoverIcon } from '../../../common';
 import { BMCFormProps } from '../Agent/types';
 import { BMCForm } from '../Agent';
 
 type AddHostModalProps = Pick<BMCFormProps, 'onClose' | 'onCreate' | 'infraEnv'> & {
   isOpen: boolean;
+  isBMPlatform: boolean;
 };
 
-const AddHostModal: React.FC<AddHostModalProps> = ({ isOpen, onClose, infraEnv, onCreate }) => {
+// TODO(mlibra): This limitation needs to be updated once https://github.com/openshift/enhancements/pull/871 lands.
+const BmOnBmOnlyPopoverIcon = () => (
+  <PopoverIcon
+    noVerticalAlign
+    bodyContent={
+      <>
+        The Advanced Cluster Manager can manage bare metal hosts when deployed on bare metal
```

Comment:
@matthewcarleton Can you please have a look at phrasing?

Reply:
@mareklibra can you share a screenshot of where this is in the UI?